ci/terraform/oidc/api-gateway.tf

```diff
@@ -946,3 +948,38 @@ resource "aws_api_gateway_integration" "orch_token_integration" {
   integration_http_method = "POST"
   uri                     = "arn:aws:apigateway:eu-west-2:lambda:path/2015-03-31/functions/arn:aws:lambda:eu-west-2:${var.orch_account_id}:function:${var.orch_token_name}:latest/invocations"
 }
+
+resource "aws_api_gateway_resource" "orch_auth_code_resource" {
+  count       = var.orch_auth_code_enabled ? 1 : 0
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
+  parent_id   = aws_api_gateway_rest_api.di_authentication_api.root_resource_id
+  path_part   = "auth-code"
+  depends_on = [
+    module.auth-code
+  ]
+}
+
+resource "aws_api_gateway_method" "orch_auth_code_method" {
+  count       = var.orch_token_enabled ? 1 : 0
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
+  resource_id = aws_api_gateway_resource.orch_token_resource[0].id
+  http_method = "GET"
+
+  depends_on = [
+    aws_api_gateway_resource.orch_auth_code_resource
+  ]
+  authorization = "NONE"
+}
+
+resource "aws_api_gateway_integration" "orch_auth_code_integration" {
+  count       = var.orch_auth_code_enabled ? 1 : 0
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
+  resource_id = aws_api_gateway_resource.orch_auth_code_resource[0].id
+  http_method = aws_api_gateway_method.orch_auth_code_method[0].http_method
+  depends_on = [
+    aws_api_gateway_resource.orch_auth_code_resource
+  ]
+  type                    = "AWS_PROXY"
+  integration_http_method = "GET"
```

Comment:
Good spot but I think this actually needs to be POST for all of them - previous version of this is [here](https://github.com/govuk-one-login/authentication-api/blob/5b820c9144773f6de6790749ea56b0f11427baf1/ci/terraform/modules/endpoint-module/api-gateway.tf#L31)

Reply:
Ah, okay, thanks.